pages/index.js

```diff
@@ -1,13 +1,65 @@
 import styled from "styled-components";
+import { useEffect, useState } from "react";
+import Image from "next/image";
+
+
+
+
```

Comment:
What are those empty code lines for? Please use prettier for formatting your code. It helps with readability. Use it in all your files before requesting a review.